inventory_management/inventory_management/report/stock_ledger/stock_ledger.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2024, Viny Selopal and contributors
+# For license information, please see license.txt
+
+import frappe
+from frappe import _
+
+def execute(filters=None):
+	columns = get_columns(filters)
+	data = get_data(filters)
+
+	return columns, data
+
+def get_columns(filters):
```

Comment:
```suggestion
def get_columns():
```
We need not pass filters here since its not used in building columns